content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -57,11 +57,20 @@ spec:
 
 ### Namespace isolation
 
-The following policy allows traffic only from the namespace `foo`. In other words, it isolates the namespace `foo` from other
-namespaces. This requires you have already enabled mTLS.
+The following two policies enable strict mTLS and allow traffic only from the namespace `foo`. In other words, it
+isolates the namespace `foo` from other namespaces.
```

Comment:
Remove the second sentence. "isolate" is misleading, unless we use deny policy which says "deny requests if it is not from 'foo' namespace". 

"allow traffic from the namespace 'foo'". -> remove "only".